docs/toc.yml

```diff
@@ -2374,84 +2374,6 @@
   - name: Walkthroughs
     href: visual-basic/walkthroughs.md
 - name: ML.NET Guide
-  href: machine-learning/index.md
```

Comment:
don't you have to delete the index.md file in this case?

Reply:
@mairaw Please review the updated version and let me know if this is still an issue. Thanks! :)